### ISSUE TITLE
Support for SSD-Mobilenet fine-tune

### ISSUE THREAD

**Issue by Leslie-Fang** [tensorflow/hub#538]
Dear tensorflow-hub developers, 
as frequently reported: https://github.com/tensorflow/models/issues/4881
The objection detection API: https://github.com/tensorflow/models/tree/master/research/object_detection is not stable.
Could we support objection detection fine-tune in tensorflowhub?

**Comment by rmothukuru** [comment]
@Leslie-Fang ,
As per my understanding, this issue is similar to #509. Can you please confirm. Thanks!

**Comment by Leslie-Fang** [comment]
@rmothukuru Thanks for the reply. Yes, similar question, I am looking for a ssd-mobilenet model trained in coco-dataset for objection detection.
I tried the https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md
But as the   https://github.com/tensorflow/models/issues/4881 reported. The loss is not stable when we fine-tune them.

**Comment by Leslie-Fang** [comment]
Close and keep an eye to https://github.com/tensorflow/hub/issues/509
